DigitalLearningSolutions.Web/Views/Home/Index.cshtml

```diff
@@ -1,16 +1,111 @@
+
+<div class="nhsuk-grid-row">
+  <div id="column" class="nhsuk-grid-column-full">
+    <h1 id="page-heading">A platform for digital learning delivery</h1>
+
+    <p id="page-subheading">Enabling health and care organisations to deliver digital learning to staff and students.</p>
+
+    <a class="nhsuk-button auth-button auth-button--left" asp-controller="Login" asp-action="Index">
+      Sign in
+    </a>
+    <a class="nhsuk-button nhsuk-button--secondary auth-button" asp-controller="Register" asp-action="Index">
+      Register
+    </a>
+    <h2 id="products">Products</h2>
+    <ul class="content-area-list">
+      <li>
+        <div class="content-area nhsuk-card">
+          <h3>Tracking System</h3>
+          <p>Distribute and track digital learning</p>
+          <ul class="content-area__sub-list">
+            <li>Provide your staff or students with access to quality assured digital learning content</li>
+            <li>Provide access to your own, hosted learning content</li>
+            <li>Comprehensive tools for managing learners and learning pathways</li>
+            <li>Track and report on learning</li>
+          </ul>
+        </div>
+      </li>
+      <li>
+        <div class="content-area nhsuk-card">
+          <h3>Content Management System</h3>
+          <p>Import and manage learning content</p>
+          <ul class="content-area__sub-list">
+            <li>Create courses, providing only necessary information</li>
+            <li>Upload digital content (tutorials, assessments, videos, documents etc) created in any tool</li>
+            <li>Structure your content to make it digestible and for ease of navigation</li>
+            <li>Publish content for your learners and even make it available to other DLS centres</li>
+          </ul>
+        </div>
+      </li>
+      <li>
+        <div class="content-area nhsuk-card">
+          <h3>Learning portal</h3>
+          <p>Access to your current, available and completed learning</p>
+          <ul class="content-area__sub-list">
+            <li>At a glance view of current learning courses and target dates</li>
+            <li>Easy, searchable access to available learning content</li>
+            <li>Quick access to completed courses and certificates</li>
+          </ul>
+        </div>
+      </li>
+    </ul>
+    <h2 id="learningcontent">Learning Content</h2>
+    <ul class="content-area-list">
+      <li>
+        <div class="content-area--with-image--image-before content-area--with-image content-area nhsuk-card nhsuk-card--clickable">
+          <div class="content-area__image-box content-area__image-box--left">
+            <img class="content-area__image" src="images/itSkillsPathwayLogo.png" alt="It Skills Pathway logo"/>
+          </div>
+          <div>
+            <h3 class="learning-content__item-text learning-content__item-header">
+              <a href="https://www.dls.nhs.uk/Learning?brand=ITSkillsPathway">

```

Comment:
I know in some of the work that I have done I have referenced the old site using the CurrentSystemBaseUrl config value in the appsettings.json files (e.g. on the /ForgotPassword/Confirm.cshtml page for the FindYourCentre link). @stellake do you know if we want to use the config value in this case, or is the href fine as it is?

Reply:
I think we should use the config value :)

Reply:
Done.